CRITICAL - 2022-04-05 09:58:25 --> Invalid file: user/login.php
#0 /Users/owner/dev/utn/isw613/codeigniter4/system/View/View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('user/login.php')
#1 /Users/owner/dev/utn/isw613/codeigniter4/system/Common.php(1089): CodeIgniter\View\View->render('user/login', Array, true)
#2 /Users/owner/dev/utn/isw613/codeigniter4/app/Controllers/User.php(14): view('user/login')
#3 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(832): App\Controllers\User->login()
#4 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#5 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /Users/owner/dev/utn/isw613/codeigniter4/public/index.php(37): CodeIgniter\CodeIgniter->run()
#7 /Users/owner/dev/utn/isw613/codeigniter4/system/Commands/Server/rewrite.php(43): require_once('/Users/owner/de...')
#8 {main}
CRITICAL - 2022-04-05 10:35:34 --> Call to a member function getAll() on null
#0 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(832): App\Controllers\News->index()
#1 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#2 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 /Users/owner/dev/utn/isw613/codeigniter4/public/index.php(37): CodeIgniter\CodeIgniter->run()
#4 /Users/owner/dev/utn/isw613/codeigniter4/system/Commands/Server/rewrite.php(43): require_once('/Users/owner/de...')
#5 {main}
CRITICAL - 2022-04-05 10:35:57 --> Invalid argument supplied for foreach()
#0 /Users/owner/dev/utn/isw613/codeigniter4/app/Views/news/index.php(8): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', '/Users/owner/de...', 8, Array)
#1 /Users/owner/dev/utn/isw613/codeigniter4/system/View/View.php(210): include('/Users/owner/de...')
#2 /Users/owner/dev/utn/isw613/codeigniter4/system/View/View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /Users/owner/dev/utn/isw613/codeigniter4/system/Common.php(1089): CodeIgniter\View\View->render('news/index', Array, true)
#4 /Users/owner/dev/utn/isw613/codeigniter4/app/Controllers/News.php(11): view('news/index')
#5 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(832): App\Controllers\News->index()
#6 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#7 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /Users/owner/dev/utn/isw613/codeigniter4/public/index.php(37): CodeIgniter\CodeIgniter->run()
#9 /Users/owner/dev/utn/isw613/codeigniter4/system/Commands/Server/rewrite.php(43): require_once('/Users/owner/de...')
#10 {main}
CRITICAL - 2022-04-05 10:37:50 --> Invalid argument supplied for foreach()
#0 /Users/owner/dev/utn/isw613/codeigniter4/app/Views/news/index.php(8): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', '/Users/owner/de...', 8, Array)
#1 /Users/owner/dev/utn/isw613/codeigniter4/system/View/View.php(210): include('/Users/owner/de...')
#2 /Users/owner/dev/utn/isw613/codeigniter4/system/View/View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /Users/owner/dev/utn/isw613/codeigniter4/system/Common.php(1089): CodeIgniter\View\View->render('news/index', Array, true)
#4 /Users/owner/dev/utn/isw613/codeigniter4/app/Controllers/News.php(11): view('news/index')
#5 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(832): App\Controllers\News->index()
#6 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#7 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /Users/owner/dev/utn/isw613/codeigniter4/public/index.php(37): CodeIgniter\CodeIgniter->run()
#9 /Users/owner/dev/utn/isw613/codeigniter4/system/Commands/Server/rewrite.php(43): require_once('/Users/owner/de...')
#10 {main}
CRITICAL - 2022-04-05 10:42:49 --> Call to undefined function App\Controllers\renderTemplate()
#0 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(832): App\Controllers\News->index()
#1 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#2 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 /Users/owner/dev/utn/isw613/codeigniter4/public/index.php(37): CodeIgniter\CodeIgniter->run()
#4 /Users/owner/dev/utn/isw613/codeigniter4/system/Commands/Server/rewrite.php(43): require_once('/Users/owner/de...')
#5 {main}
ERROR - 2022-04-05 11:07:44 --> Error connecting to the database: No such file or directory
ERROR - 2022-04-05 11:08:24 --> Error connecting to the database: No such file or directory
ERROR - 2022-04-05 11:16:27 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-04-05 11:16:27 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 /Users/owner/dev/utn/isw613/codeigniter4/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /Users/owner/dev/utn/isw613/codeigniter4/system/Database/BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /Users/owner/dev/utn/isw613/codeigniter4/system/Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 /Users/owner/dev/utn/isw613/codeigniter4/system/BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 /Users/owner/dev/utn/isw613/codeigniter4/app/Controllers/News.php(11): CodeIgniter\BaseModel->findAll()
#5 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(832): App\Controllers\News->index()
#6 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#7 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /Users/owner/dev/utn/isw613/codeigniter4/public/index.php(37): CodeIgniter\CodeIgniter->run()
#9 /Users/owner/dev/utn/isw613/codeigniter4/system/Commands/Server/rewrite.php(43): require_once('/Users/owner/de...')
#10 {main}
ERROR - 2022-04-05 11:16:45 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-04-05 11:16:45 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 /Users/owner/dev/utn/isw613/codeigniter4/system/Database/BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 /Users/owner/dev/utn/isw613/codeigniter4/system/Database/BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 /Users/owner/dev/utn/isw613/codeigniter4/system/Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 /Users/owner/dev/utn/isw613/codeigniter4/system/BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 /Users/owner/dev/utn/isw613/codeigniter4/app/Controllers/News.php(11): CodeIgniter\BaseModel->findAll()
#5 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(832): App\Controllers\News->index()
#6 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#7 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /Users/owner/dev/utn/isw613/codeigniter4/public/index.php(37): CodeIgniter\CodeIgniter->run()
#9 /Users/owner/dev/utn/isw613/codeigniter4/system/Commands/Server/rewrite.php(43): require_once('/Users/owner/de...')
#10 {main}
ERROR - 2022-04-05 11:17:23 --> Table 'newscover.newssources' doesn't exist
CRITICAL - 2022-04-05 11:17:23 --> Call to a member function getResult() on bool
#0 /Users/owner/dev/utn/isw613/codeigniter4/system/BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 /Users/owner/dev/utn/isw613/codeigniter4/app/Controllers/News.php(11): CodeIgniter\BaseModel->findAll()
#2 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(832): App\Controllers\News->index()
#3 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#4 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /Users/owner/dev/utn/isw613/codeigniter4/public/index.php(37): CodeIgniter\CodeIgniter->run()
#6 /Users/owner/dev/utn/isw613/codeigniter4/system/Commands/Server/rewrite.php(43): require_once('/Users/owner/de...')
#7 {main}
ERROR - 2022-04-05 11:18:14 --> Table 'newscover.newssources' doesn't exist
CRITICAL - 2022-04-05 11:18:14 --> Call to a member function getResult() on bool
#0 /Users/owner/dev/utn/isw613/codeigniter4/system/BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 /Users/owner/dev/utn/isw613/codeigniter4/app/Controllers/News.php(11): CodeIgniter\BaseModel->findAll()
#2 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(832): App\Controllers\News->index()
#3 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#4 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /Users/owner/dev/utn/isw613/codeigniter4/public/index.php(37): CodeIgniter\CodeIgniter->run()
#6 /Users/owner/dev/utn/isw613/codeigniter4/system/Commands/Server/rewrite.php(43): require_once('/Users/owner/de...')
#7 {main}
ERROR - 2022-04-05 11:25:19 --> Table 'newscover.newssources' doesn't exist
CRITICAL - 2022-04-05 11:25:19 --> Call to a member function getResult() on bool
#0 /Users/owner/dev/utn/isw613/codeigniter4/system/BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 /Users/owner/dev/utn/isw613/codeigniter4/app/Controllers/News.php(11): CodeIgniter\BaseModel->findAll()
#2 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(832): App\Controllers\News->index()
#3 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#4 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /Users/owner/dev/utn/isw613/codeigniter4/public/index.php(37): CodeIgniter\CodeIgniter->run()
#6 /Users/owner/dev/utn/isw613/codeigniter4/system/Commands/Server/rewrite.php(43): require_once('/Users/owner/de...')
#7 {main}
ERROR - 2022-04-05 11:25:25 --> Table 'newscover.newssources' doesn't exist
CRITICAL - 2022-04-05 11:25:25 --> Call to a member function getResult() on bool
#0 /Users/owner/dev/utn/isw613/codeigniter4/system/BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 /Users/owner/dev/utn/isw613/codeigniter4/app/Controllers/News.php(11): CodeIgniter\BaseModel->findAll()
#2 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(832): App\Controllers\News->index()
#3 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#4 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /Users/owner/dev/utn/isw613/codeigniter4/public/index.php(37): CodeIgniter\CodeIgniter->run()
#6 /Users/owner/dev/utn/isw613/codeigniter4/system/Commands/Server/rewrite.php(43): require_once('/Users/owner/de...')
#7 {main}
ERROR - 2022-04-05 11:25:35 --> Table 'newscover.newssources' doesn't exist
CRITICAL - 2022-04-05 11:25:35 --> Call to a member function getResult() on bool
#0 /Users/owner/dev/utn/isw613/codeigniter4/system/BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 /Users/owner/dev/utn/isw613/codeigniter4/app/Controllers/News.php(11): CodeIgniter\BaseModel->findAll()
#2 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(832): App\Controllers\News->index()
#3 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#4 /Users/owner/dev/utn/isw613/codeigniter4/system/CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /Users/owner/dev/utn/isw613/codeigniter4/public/index.php(37): CodeIgniter\CodeIgniter->run()
#6 /Users/owner/dev/utn/isw613/codeigniter4/system/Commands/Server/rewrite.php(43): require_once('/Users/owner/de...')
#7 {main}
